packages/web-config-server/src/apiV1/dataBuilders/helpers/calculateOperationForAnalytics.js

```diff
@@ -63,35 +115,87 @@ const sumDataValues = (analytics, dataValues, filter = {}) => {
   return sum;
 };
 
-const countDataValues = (analytics, dataValues, filter, config) => {
+const countDataValues = async (analytics, dataValues, filter, config, models) => {
   return sumDataValues(
     analytics.map(a => ({
       ...a,
       value: checkValueSatisfiesCondition(a.value, config?.countCondition || '*') ? 1 : 0,
     })),
     dataValues,
     filter,
+    models,
   );
 };
 
+const buildOrganisationUnitParentValue = async (analytics, models) => {
```

Comment:
In general, feel free to use `orgUnit` instead of `organisationUnit` - common abbreviations help making the code simpler, and we have decided that we prefer `orgUnit` over the full version

```suggestion
const buildOrgUnitParentMapForAnalytics = async (analytics, models) => {
```